### PR TITLE
tree-wide: use error in xxx_from_string()

### PR DESCRIPTION
**Pull request by yuwata**
Follow-ups for #11484.

### CODE REVIEW

File: src/network/networkd-lldp-tx.c [hunk 1]
```diff
@@ -491,3 +452,13 @@ int config_parse_lldp_mud(
 
         return free_and_replace(n->lldp_mud, unescaped);
 }
+
+static const char * const lldp_emit_table[_LLDP_EMIT_MAX] = {
+        [LLDP_EMIT_NO] = "no",
+        [LLDP_EMIT_NEAREST_BRIDGE] = "nearest-bridge",
+        [LLDP_EMIT_NON_TPMR_BRIDGE] = "non-tpmr-bridge",
+        [LLDP_EMIT_CUSTOMER_BRIDGE] = "customer-bridge",
```
Comment (keszybz) [inline src/network/networkd-lldp-tx.c:460]: Align at "=" ?

**Review by keszybz — COMMENTED** [review]
LGTM, but ci is unhappy.

Various bionic-* tests fail with:
`subprocess.CalledProcessError: Command '['systemctl', 'set-default', 'graphical.target']' returned non-zero exit status 1.`
in boot-and-services.
I'll restart all of them to verify that it's not a transient issue.

**Comment by keszybz** [comment]
All ubuntu ci failed again.

**Comment by yuwata** [comment]
@keszybz Thank you for your review. Updated. Now the change for `free_and_strdup_warn()` is dropped, i.e., it returns 0 on success. `systemctl set-default` propagates the return value of `free_and_strdup_warn()`, and it uses `DEFINE_MAIN_FUNCTION_WITH_POSITIVE_FAILURE()` macro, so the ubuntu CIs were failed.

File: src/debug-generator/debug-generator.c [hunk 2]
```diff
@@ -68,27 +68,21 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
...
                 const char *target;
 
                 target = runlevel_to_target(key);
-                if (target) {
-                        r = free_and_strdup(&arg_default_unit, target);
-                        if (r < 0)
-                                return log_error_errno(r, "Failed to set default unit %s: %m", target);
-                }
+                if (target)
+                        return free_and_strdup(&arg_default_unit, target);
```
Comment (dnicolodi) [inline src/debug-generator/debug-generator.c]: `free_and_strdup_warn`?

Comment (yuwata) [reply src/debug-generator/debug-generator.c]: Ouch! Thank you!

**Pull request merged**